Components/Lottie.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import LottieView from 'lottie-react-native'; //for mobile view 
+//import LottieView from 'react-native-web-lottie'; //for web view
+import LottieAnimation from '../assets/animations/loader-movie.json';
+import { LottieContainer } from '../styled-components/styles';
+
+const Lottie = () => {
+	return (
+		<LottieContainer>
```

Comment:
Ups, I think you have changed size of one tab (regarding indentation) over here, tab as 2 spaces is completely enough